types/jsreport__nodejs-client/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for @jsreport/nodejs-client 3.0
+// Project: https://github.com/jsreport/jsreport/tree/master/packages/nodejs-client
+// Definitions by: reediculous456 <https://github.com/reediculous456>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import JsReport = require('jsreport');
+import { ServerResponse } from 'http';
+
+declare namespace JsReportNodeJsClient {
+    interface ClientRenderResponse extends ServerResponse {
+        body(): Promise<Buffer>;
+    }
+
+    interface Client {
+        render(req: Partial<JsReport.Request>, options?: object): Promise<ClientRenderResponse>;
+    }
+}
+
+declare function CreateJsReportClient(url: string, username?: string, password?: string): JsReportNodeJsClient.Client;
+export = CreateJsReportClient;
```

Comment:
```suggestion
declare function createJsReportClient(url: string, username: string, password: string): JsReport.Client;
declare function createJsReportClient(url: string): JsReport.Client;

export = createJsReportClient;
```
I understand this are types for a aliased/fork package. But that method can be overloaded, as per usage. CamelCase is not a JS convention for functions that are not constructor type, and this is irrelevant to usage